data/simpy.gram

```diff
@@ -183,6 +193,7 @@ full_expression: NAME ':=' expression | expression
 yield_expression: yield_expr | expression
 expression[expr_ty]:
     | lambdef
+    | yield_expr
```

Comment:
This doesn't look right, `yield_expr` is also in `yield_expression` one line above. Maybe that `yield_stmt` existed for a reason? Also a lambda expression is *not* allowed to contain an unparenthesized yield expression.

Reply:
I thought that this would be okay, since `simpy_cpython` still succeeds. Do you think this would be better?

```
small_stmt[stmt_ty]:
    | yield_stmt

yield_stmt[stmt_ty]: a=yield_expr { _Py_Expr(a, EXTRA_EXPR(a, a)) }
```

Reply:
Yeah, that should work.

The reason `make simpy_cpython` doesn't fail is that there are no negative tests. Try this:
```
lambda: yield
```
That is an error in regular Python, but the change you propose in this PR would make it pass. (To get the same effect in regular Python, use `lambda: (yield)`.)

Reply:
Got it.